templates/component.stories.tsx.hbs

```diff
@@ -0,0 +1,20 @@
+import { Meta } from "@storybook/react";
+
+import { {{pascalCase name}} } from "@/lib/sections/{{pascalCase name}}/{{pascalCase name}}";
```

Comment:
This line assumes that all created components are stored in the sections folder, as a result, it errors when component type elements are created

Reply:
Good spot! Thanks, will fix.